build-tests/x86/fedora/test-image-live-disk/appliance.kiwi

```diff
@@ -30,7 +30,12 @@
         </type>
     </preferences>
     <preferences profiles="Virtual">
-        <type image="oem" filesystem="ext4" kernelcmdline="console=ttyS0" firmware="uefi" format="qcow2">
+        <type image="oem" filesystem="btrfs" kernelcmdline="console=ttyS0" firmware="uefi" format="qcow2" btrfs_create_toplevel_subvolume="true" rootfs_label="fedora">
+            <systemdisk name="fedora">
+                <volume name="@root=root"/>
+                <volume name="home" mountpoint="/home" parent="/"/>
+                <volume name="machines" mountpoint="/var/lib"/>
```

Comment:
I'm confused by this. Shouldn't this be `<volume name="machines" parent="/var/lib"/>`. As a nested volume, it'll automount anyway.

Reply:
You confuse me, you suggested that change in your review and I just adapted. So what do you want to achieve ?

Reply:
done, I kept the mountpoint though, it's easier to read and all this is confusing to no end anyway because fedora does it completely different than suse and all this should not break. No fun to be honest